### PR TITLE
feat(xterm): implementation of xterm terminal

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

File: src/app/styles/terminal.sass [hunk 1]
```diff
@@ -1,13 +1,184 @@
...
   background: #000000
-  height: 600px
-  -webkit-font-smoothing: auto
+  height: 700px
+  width: 100%
+  font-family: Monaco, monospace
+  font-size: 12px
+
+  span
+    color: #f8f8f2
```
Comment (jkuri) [inline src/app/styles/terminal.sass]: please use white color here `#ffffff`.

File: src/app/styles/terminal.sass [hunk 2]
```diff
@@ -1,13 +1,184 @@
...
+  display: inline-block
+  color: #ffffff !important
   background: #000000
-  height: 600px
-  -webkit-font-smoothing: auto
+  height: 700px
+  width: 100%
+  font-family: Monaco, monospace
+  font-size: 12px
+
```
Comment (jkuri) [inline src/app/styles/terminal.sass]: add `padding: 10px` here please, it looks better.

File: src/app/styles/terminal.sass (hunk 1 above)
Comment (jkuri) [reply src/app/styles/terminal.sass]: `padding: 5px 10px;` looks good.

File: src/app/styles/terminal.sass (hunk 2 above)
Comment (jkuri) [reply src/app/styles/terminal.sass]: `padding: 5px 10px;` looks good.

File: src/app/styles/terminal.sass (hunk 1 above)
Comment (Izak88) [reply src/app/styles/terminal.sass]: done

File: src/app/styles/terminal.sass (hunk 2 above)
Comment (Izak88) [reply src/app/styles/terminal.sass]: done

**Pull request merged**